bip-csfs.md

```diff
@@ -0,0 +1,122 @@
+| BIP-XXXX   | `CHECKSIGFROMSTACK`
+| :--------- | :------------------
+| Layer      | Consensus (soft fork)
+| Author     | Brandon Black <freedom@reardencode.com>, Jeremy Rubin <j@rubin.io>
+| Discussion | https://delvingbitcoin.org/t/lnhance-bips-and-implementation/376/9
+| Status     | Draft
+| Type       | Standards Track
+| Created    | 2023-12-22
+| License    | BSD-3-CLAUSE
```

Comment:
Please use the [preamble format](https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#bip-header-preamble) specified in BIP2.